ui/StatusQ/src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -368,12 +373,20 @@ Item {
                         wrapMode: root.multiline ? Text.WrapAtWordBoundaryOrAnywhere : TextEdit.NoWrap
 
                         Keys.onReturnPressed: {
-                            root.keyPressed(event)
                             event.accepted = !multiline && !acceptReturn
+                            // Special case for single line inputs, where we want to accept the return key, but notify the parend
```

Comment:
The enter/return key press is signalled twice. Found while adding the qml tests for the EnterSeedPhrase component